python/triqs/lattice/utils.py

```diff
@@ -246,3 +246,161 @@ def TB_from_pythTB(ptb):
                     orbital_names=[str(i) for i in range(ptb.get_num_orbitals())])
 
     return TBL
+
+# importing necessary dependencies
+from itertools import product as itp
+from pythtb import *
+# from triqs.lattice.tight_binding import TBLattice # TRIQS needs to be installed for this dependency to work
+import sympy as sp
+from sympy import *
+import warnings
+
+def sympyfy(tb_lat_obj, analytical = True):
```

Comment:
I am not sure what is the meaning of the numerical form? Just the Fourier transform of H(R)? On what kind of mesh?